tests/pdifftests/test.py

```diff
@@ -3,10 +3,8 @@
 from selenium.webdriver.support.ui import Select
 from selenium.webdriver.common.action_chains import ActionChains
 
```

Comment:
This is a massive silly nit, so don't worry about changing it, but in general 2 newlines between imports and top-level definitions.